src/common/util.c

```diff
@@ -47,7 +47,12 @@
 #endif
 
 #define PATH_SEPARATOR '/'
+
+#if defined(OS_FREEBSD)
+#define OSCAP_PCRE_EXEC_RECURSION_LIMIT_DEFAULT 3500
```

Comment:
How would you justify this exception? We choose 5000 pretty arbitrary, it might make sense to lower it for all platforms, if you have some evidence that 3500 is better.

Reply:
Openscap would crash on FreeBSD when running `textfilecontent54/test_recursion_limit.sh` with a recursion limit of 4k or higher, so I chose 3.5k fairly arbitrarily after that point.

Reply:
Oh, good. Let's change it to 3500 for everything then.